src/components/MultiSelect/MultiSelect.stories.tsx

```diff
@@ -271,6 +272,26 @@ export const withControls: Story = {
     },
     jest: ["MultiSelect.test.tsx"],
   },
+  play: async ({ canvasElement }) => {
```

Comment:
It would be good to add some more expectations.  For example, verifying that no checkboxes are checked after clicking the clear button (the selected options counter).  Or verifying that the dropdown is closed after clicking the `esc` key.  Or verifying that the selected options counter is visible after selecting at least one option.  Things like that.